phpBB/includes/functions.php

```diff
@@ -2851,10 +2851,13 @@ function get_preg_expression($mode)
 		// Whoa these look impressive!
 		// The code to generate the following two regular expressions which match valid IPv4/IPv6 addresses
 		// can be found in the develop directory
+
+		// @deprecated
```

Comment:
I think there isn't that much of a point to deprecate these. The preg expressions are still valid and might also be used by extensions.

Reply:
It have no sense to keep that when php already do that natively with filter_var